eslint-custom-rules/queries-must-be-in-queries-folder.test.js

```diff
@@ -0,0 +1,57 @@
+const { RuleTester } = require('eslint')
+const rule = require('./queries-must-be-in-queries-folder')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+})
+
+const mockContext = {
+  getFilename: () => '',
+  getSourceCode: () => ({ ast: {} }),
+}
+
+const createValidTest = (filename) => ({
+  code: '// test',
+  filename,
+  errors: [],
+})
+
+const createInvalidTest = (filename) => ({
+  code: '// test',
+  filename,
+  errors: [{ messageId: 'mustBeInQueriesFolder' }],
+})
+
+const validFilenames = [
+  '/src/queries/useGetDataQuery.js',
+  '/src/queries/nested/folder/useGetDataQuery.js',
+  '/src/queries/useUpdateDataMutation.js',
+  '/src/queries/nested/folder/useUpdateDataMutation.js',
+  '/src/components/Button.js',
+  '/src/queries/Button.js',
+  '/src/queries/useGetDataQuery.ts',
+  '/src/queries/nested/folder/useGetDataQuery.ts',
```

Comment:
il y a un cas nested, c'est chouette, on pourrait montrer un autre cas qu'on aura très fréquemment dans la codebase : le dossier queries qui n'est pas directement dans src

```suggestion
  '/src/features/offers/queries/useGetDataQuery.ts',
```